client/src/App.js

```diff
@@ -16,42 +16,43 @@ import ResetPassword from './pages/ResetPassword';
 
 const App = () => {
   return (
-    <Provider store={store}>
-      <PersistGate loading={null} persistor={persistor}>
-        <Router>
-          <Header />
-          <div className="container">
-            <Switch>
-              <Route path="/" exact component={() => <Home />} />
-              <Route
-                path="/activate/:id"
-                component={() => <ActivateAccount />}
-              />
+    <div className="min-vh-100 d-flex flex-column">
+      <Provider store={store}>
+        <PersistGate loading={null} persistor={persistor}>
+          <Router>
```

Comment:
Given the fact that this Router grew over time, I believe it would be a simple refactor to move it to another file called navigation and import it here also for code clarity.

Reply:
Fixed